test/service/test_response.py

```diff
@@ -2342,6 +2348,76 @@ def test_inner_entity_samples(self):
                 self.assertEqual(expected_hits, [hit['samples'] for hit in hits])
 
 
+@patch_dss_endpoint
+@patch_source_cache
+class TestSchemaTestDataCannedBundle(WebServiceTestCase):
+    maxDiff = None
+
+    @classmethod
+    def bundles(cls) -> List[BundleFQID]:
+        return [
+            # Bundles from the canned staging area, both for project 90bf705c
+            # https://github.com/HumanCellAtlas/schema-test-data/
+            cls.bundle_fqid(uuid='4da04038-adab-59a9-b6c4-3a61242cc972',
+                            version='2021-01-01T00:00:00.000000Z'),
+            cls.bundle_fqid(uuid='d7b8cbff-aee9-5a05-a4a1-d8f4e720aee7',
+                            version='2021-01-01T00:00:00.000000Z'),
+        ]
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls._setup_indices()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._teardown_indices()
+        super().tearDownClass()
+
+    def test_project_cell_count(self):
+        """
+        Verify the project 'estimatedCellCount' value across the various endpoints
+        """
+        expected_cell_counts = {
+            'files': [10000, 10000],
+            'samples': [10000, 10000],
+            'projects': [10000],
+            'bundles': [10000, 10000],
+        }
+        actual_cell_counts = {}
+        params = {'catalog': self.catalog}
+        for entity_type in expected_cell_counts.keys():
+            with self.subTest(entity_type=entity_type):
```

Comment:
Test output is going to look confusing if the assertion is outside the sub test.